test/verify/check-multi-machine

```diff
@@ -504,6 +504,7 @@ class TestMultiMachine(MachineCase):
         b.enter_page("/dashboard")
         # Check the server image
         b.wait_present("#dashboard-hosts a[data-address='%s']" % m2.address)
+        b.wait_present("#dashboard-hosts a[data-address='%s'] img" % m2.address)
```

Comment:
Oh interesting, it errors out before the timeout if the leading elements aren't present? Good to know for writing tests myself, thanks.